ambari-server/src/main/python/bootstrap.py

```diff
@@ -107,8 +107,9 @@ def run(self):
     scpstat = subprocess.Popen(scpcommand, stdout=subprocess.PIPE,
                                stderr=subprocess.PIPE)
     log = scpstat.communicate()
-    errorMsg = log[1]
-    log = log[0] + "\n" + log[1]
+    log_value0 = log[0].decode()
```

Comment:
Can you add the check if it is instance of bytes?

Reply:
> Can you add the check if it is instance of bytes?

In python3, Subprocesses output bytes, not characters when calling `communicate()`, so we need to add `universal_newlines=True` if we want str.

https://docs.python.org/3.9/library/subprocess.html